client/app/main/dashboard/dashboard.controller.js

```diff
@@ -37,7 +126,8 @@ function DashboardCtrl($scope, $timeout, AbsenceRecord, GridDefaults, Student) {
         $scope.loading = false;
         $timeout($scope.gridApi.treeBase.expandAllRows);
       });
-      $scope.menuItems.length = 1;
+      $scope.menuItems.length = 0;
+      [].push.apply($scope.menuItems, defaultMenuItems);
```

Comment:
I get the basics of commands like `call` and `apply`, any other resources/tutorials that you can recommend on this subject?